src/main/java/com/synopsys/integration/detect/tool/signaturescanner/operation/CreateScanBatchRunnerWithBlackDuck.java

```diff
@@ -48,13 +48,14 @@ public ScanBatchRunner createScanBatchRunner(BlackDuckServerConfig blackDuckServ
 
         ScannerInstaller scannerInstallerVariant;
 
-        if (shouldUseNewApiScannerInstaller(blackDuckVersion)) {
+        if (shouldUseToolsApiScannerInstaller(blackDuckVersion)) {
             logger.trace("Using new Scan CLI download API.");
             scannerInstallerVariant = new ToolsApiScannerInstaller(
                     slf4jIntLogger,
                     blackDuckHttpClient,
                     cleanupZipExpander,
                     scanPathsUtility,
+                    keyStoreHelper,
```

Comment:
This is the line of importance, the rest as you said is just version bump + method name update. @andrian-sevastyanov